python-package/lightgbm/engine.py

```diff
@@ -370,6 +370,8 @@ def cv(params, train_set, num_boost_round=10,
 
     if metrics:
         params.setdefault('metric', [])
+        if isinstance(params['metric'], string_type):
```

Comment:
User set metrics in cv for evaluate each iterator is better or worse.
set metrics in param  for loss function.